packages/web-components/fast-foundation/src/storybook-typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.html" {
```

Comment:
I think we can remove the storybook typing if we're not using it. I also saw mention it in some of the "." prefixed files in the root of the package.

Reply:
yes, indeed will remove

Reply:
I believe I've pulled all references (including the now-unnecessary tsconfig.build.json file)